solr/example/files/README.md

```diff
@@ -111,38 +120,46 @@ For further explanations, see the frequently asked questions at the end of the g
 * Another way to query the index is by manipulating the URL in your address bar once in the browse view.
 
 * i.e. : [http://localhost:8983/solr/files/browse?q=Lucene](http://localhost:8983/solr/files/browse?q=Lucene)
-<hr>
-##FAQs
+
+## FAQs
 
 * Why use -d when creating a core?
 	* -d specifies a specific configuration to use.  This example as a configuration tuned for indexing and query rich
 	  text files.
 	
 * How do I delete a core?
-	* To delete a core (i.e. files), you can enter the following in your command shell:
-		bin/solr delete -c files
-
-	* You should see the following output:
+  * To delete a core (i.e. files), you can enter the following in your command shell:
 	
-		Deleting core 'files' using command:
-			http://localhost:8983/solr/admin/cores?action=UNLOAD&core=files&deleteIndex=true&deleteDataDir=true&deleteInstanceDir=true
+    ```
+    bin/solr delete -c files
+    ```
+ 
+  * You should see the following output:
+	
+	Deleting core 'files' using command:
+		
+	```
+	http://localhost:8983/solr/admin/cores?action=UNLOAD&core=files&deleteIndex=true&deleteDataDir=true&deleteInstanceDir=true
     
-			{"responseHeader":{
-					"status":0,
-					"QTime":19}}
-
-	* This calls the Solr core admin handler, "UNLOAD", and the parameters "deleteDataDir" and "deleteInstanceDir" to ensure that all data associated with core is also removed
+	{"responseHeader":{
+		"status":0,
+		"QTime":19}}
+    ```
+  
+  * This calls the Solr core admin handler, "UNLOAD", and the parameters "deleteDataDir" and "deleteInstanceDir" to ensure that all data associated with core is also removed
 
 * How can I change the /browse UI?
 
-	The primary templates are under example/files/conf/velocity.  **In order to edit those files in place (without having to
-	re-create or patch a core/collection with an updated configuration)**, Solr can be started with a special system property
-	set to the _absolute_ path to the conf/velocity directory, like this: 
-	
-		bin/solr start -Dvelocity.template.base.dir=</full/path/to>/example/files/conf/velocity/
+    The primary templates are under example/files/conf/velocity.  **In order to edit those files in place (without having to
+    re-create or patch a core/collection with an updated configuration)**, Solr can be started with a special system property
+    set to the _absolute_ path to the conf/velocity directory, like this: 
 	
-        If you want to adjust the browse templates for an existing collection, edit the core’s configuration
-        under server/solr/files/conf/velocity.
+    ```
+    bin/solr start -Dvelocity.template.base.dir=</full/path/to>/example/files/conf/velocity/
+    ```
+ 
+If you want to adjust the browse templates for an existing collection, edit the core’s configuration
+under server/solr/files/conf/velocity.
 
 
 =======
```

Comment:
At least on GitHub, this isn't showing as markup.  (I assume this line is at the end directly about the provenance info)

Reply:
Thanks, I missed it. It was fixed as this: https://github.com/mocobeta/lucene-solr/tree/solr-change-txt-to-md/solr/example/files